oligo_designer_toolsuite/oligo_transcript_generation/_gene_transcript.py

```diff
@@ -23,17 +24,19 @@ def __init__(self, file_sequence, file_annotation):
             self.file_annotation
         )  # dataframe with annotation file
 
-    def generate_for_reference(self, block_size, dir_output="./output/annotation"):
+    def generate_for_reference(
+        self, block_size, file_gene_transcript_fasta, dir_out="output/annotation"
```

Comment:
we should not call it `file_gene_transcript_fasta` because the background dB can be something else than transcripts